plugins/builddep.py

```diff
@@ -174,8 +174,9 @@ def _src_deps(self, src_fn):
     def _spec_deps(self, spec_fn):
         try:
             spec = rpm.spec(spec_fn)
-        except ValueError:
-            msg = _("Failed to open: '%s', not a valid spec file.") % spec_fn
+        except ValueError as ex:
+            msg = _("Failed to open: '%s', not a valid spec file: %s") % (
+                    spec_fn, ex)
```

Comment:
What about to add:
```from dnf.i18n import ucd```
and
```spec_fn, ucd(ex)```
Just for case that something strange is in the spec file.